### ISSUE TITLE
'Open folder' of download, in Firefox: file gets run instead

### ISSUE THREAD

**Issue by CottonEaster** [linuxmint/nemo#1893]
```
 * Nemo version (nemo --version): 3.8.4
 * Is issue with desktop or windowed nemo? Unclear
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint 19
 * 32 or 64 bit: 64
 ```

**Issue**

I am unsure whether this is a Mint bug (and, if it is, whether it is a nemo bug) but, when Firefox downloads a .deb or a .gz file, and then tells Firefox to open the folder - *not* to open the file - it opens the file. That is, in the case of the .deb file, gdebi runs, and in the case of the archive, file-roller runs.

**Steps to reproduce**

Download .deb or .gz in Firefox. Tell Firefox to open the folder into which the download went.

**Expected behaviour**

The folder should open.

**Other information**

Firefox 61.0

**Comment by leigh123linux** [comment]
Are you using firefox to 'save file' or 'open with *' ?, 
The issue doesn't reproduce here.

**Comment by CottonEaster** [comment]
Let me spell out the steps.

1. Click a download link in Firefox, and, when asked whether one wants to save or open, choose save. Screenshot:

![image](https://user-images.githubusercontent.com/9998846/42437500-33f29720-8355-11e8-8a8a-c738ea70cead.png)

2. When the download is complete, click the Firefox toolbar save icon and click the folder icon - screenshot:
![image](https://user-images.githubusercontent.com/9998846/42437571-699f5fde-8355-11e8-92c2-59db8156db8c.png)

Result (screenshot):

![image](https://user-images.githubusercontent.com/9998846/42437632-93eef1f0-8355-11e8-8749-13da9a8989c3.png)

**Comment by JosephMcc** [comment]
Can't reproduce here either. Tried with two different debs and open containing folder does exactly what it says

**Comment by CottonEaster** [comment]
Thanks.

I should perhaps add that my default file manager is PcManFM. Cf. also, perhaps, #1589.

**Comment by CottonEaster** [comment]
Aha - the problem does not occur when Nemo is the default file manager.

**Comment by JosephMcc** [comment]
If you are going to open so many issues, please at least provide the relevant info so we aren't wasting our time trying to reproduce things on setups that aren't even close to yours. You have a problem with Firefox, somehow blame it on Nemo and then later decide to let us know that Nemo isn't even being used.

**Comment by CottonEaster** [comment]
I take your point, but some things can be said in my defence.

I should indeed have provided more information (though I confess to doing this sometimes: waiting to see whether a report is acted upon before providing more information; in the past - much less so now - it seemed that some reports I filed were ignored; I can see, though, that there is a circularity to my reasoning here).

I did say I was unsure Nemo was to blame (and sometimes it seems unclear where to file bug reports, partly because a few of the repositories provide little by way of description). Indeed I said also that I was unsure that Mint was to blame at all. But in reporting the problem I have to start somewhere - either with Mint, or Firefox, or PcManFM. I decided on Mint, and can cite the other bug report that I mentioned (#1589) as some degree of warrant for that. (Also, if you were to suggest that I should not use PcManFm, then I would point to #1855, #1378, and #1683.)

**In short**: _it may well be a Mint problem and I did not insist that it was; but I should have said more initially_.

**Comment by clefebvre** [comment]
You're opening the issue too soon. You should do so after troubleshooting and when the cause is known. You're at the observations phase here at the time of opening the issue, that pushes devs either to arbitrarily close your issues or take part in the troubleshooting with you... and although that might lead to an actual bug, their time is much better used at fixing bugs which are already identified (it's not like there aren't any).

http://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/

**Comment by CottonEaster** [comment]
Thank you, Clem. I shall investigate further and if necessary submit a fuller, new bug report somewhere - but, where? (please advise) - on the Mint site.
